### ISSUE TITLE
[Question] how to avoid warning when using the function envs.call()

### ISSUE THREAD

**Issue by yinfanyi** [Farama-Foundation/Gymnasium#1089]
### Question

The usage of `envs.call` is "Call a method, or get a property, from each parallel environment". 

However, when I use this function, a warning is returned: 
WARN: env.model_path to get variables from other wrappers is deprecated and will be removed in v1.0, to get this variable you can do `env.unwrapped.xxx` for environment variables or `env.get_wrapper_attr('xxx')` that will search the reminding wrappers.

Could you tell me how to avoid this warning?

**Comment by yinfanyi** [comment]
Additions: My parallel environment includes several mujoco environments, and each mujoco environment has a model_path 
 which definition is as follows:
```
class TestMujocoEnv(MujocoEnv, utils.EzPickle):
    def __init__(self, model_path,  **kwargs):
        observation_space = Box(low=-np.inf, high=np.inf, shape=(8,), dtype=np.float64)
        MujocoEnv.__init__(
            self,
            model_path=model_path,
            frame_skip=50,
            observation_space=observation_space,
            default_camera_config=DEFAULT_CAMERA_CONFIG,
            **kwargs,
        )
        utils.EzPickle.__init__(self, **kwargs)
        self.model_path = model_path    
       ...
```
then I want to use `envs.call(name='model_path')`. It can return every single environment's model_path, but with a warning.

**Comment by pseudo-rnd-thoughts** [comment]
That warning is for accessing single environment attributes so for using Async or Sync vector environment, you can't do anything but this issue will be fixed behind the scenes in v1.0
You can try it out with `pip install gymnasium==1.0.0a2`

**Comment by yinfanyi** [comment]
thanks!
